content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+
+If tech companies worries sincerely about their users' safety, they need to stop collecting our data now, and to not have it when the warrants arrive. At least they must collect less data, encrypt it, not share it with dubious actors, not force users to authenticate, etc. Of course this will negatively impact their business model but maybe it is time for them to change their retribution model. Society is evolving, they must also evolve if they have consideration for their customers.
+
+#### Smartphones and Apps
+
+Another side of this situation is smartphone apps. Some may have guessed the level of knowledge of social media on their users, but maybe some forgot that apps on their smartphone are collecting a lot of sensible information. Again, some apps may use some tracker from facebook that are hidden in some freeware libraries and frameworks they include in their app to save development time. Facebook and Google are everywhere. Android smartphones are at risk but Apple’s smartphones are also at risk even if less exposed.
+
+One sensible type of App that was heavily discussed regarding the abortion law was menstrual cycle tracking apps (55 million active users). Highly appreciated by women for tracking their sexual health and their fertility, their data stored in the cloud may be shared to a moving and huge list of partners. This data sharing is, of course, a source of revenue for these apps that may be free or have free plans. Paying is also not a guarantee of confidentiality, but it shall be very clear in users’ mind that “when it's free, you are the product”… In the U.S., health-related information collected by apps is not inherently protected by any law, HIPAA law only protects information collected and owned by a medical provider. A lot of companies are willing to pay huge amounts of money for this very intimate data. It could be for marketing (selling services, health devices, or any innovative use). Of course, as said before, state police could request this data. Menstrual cycle can show that a woman is pregnant, for how long, and can also reveal that suddenly she is not pregnant anymore, possibly violating the state’s law.
+
+In this context, many american women started deleting their apps and requesting past data erasure. But who can certify that past data is fully erased and impossible to undelete? Some women started to use the Clue app that is based in E.U. (Berlin) thinking that GDPR is protecting them. Unfortunately, GDPR here is not really a protection because of what we already flagged above: firstly even a E.U. company has to respond positively to a legitimate legal request from US authorities, secondly, the storage of an E.U. app may not be simple and may use servers located outside Europe. Here again there is a big risk that state police may request this data.
```

Comment:
Why talk about past data? Can't this simply talk about erasing your data without specifying "past", for a matter of simplicity?

Reply:
Also, see remarks in the French part about the legal part.

Reply:
Removed the "past" part.